lib/repo/git_repository.rb

```diff
@@ -497,25 +605,38 @@ def self.add_user(user_id, permissions, repo_name)
 
       # Adds a user with given permissions to the repository
       if !File.exist?(Repository.conf[:REPOSITORY_PERMISSION_FILE])
-        File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w").close() # create file if not existent
+        # create file if not existent
+        File.open(Repository.conf[:REPOSITORY_PERMISSION_FILE], "w").close()
```

Comment:
Do not use parentheses for method calls with no arguments.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.